app/src/components/games/crescendo/CrescendoAutoViewer.tsx

```diff
@@ -1,8 +1,7 @@
-import React from 'react';
 import {View, Text} from 'react-native';
```

Comment:
remember to reformat file so that it is consistant formatting-wise with spacing after the braces

Reply:
and also the quotation marks